server/schemas/queries/login.js

```diff
@@ -0,0 +1,14 @@
+const { User } = require('../../models/User');
```

Comment:
Change filename to user.js and use 'user' throughout files in place of 'login'. We have a login mutation so we which keep this query resolve name distinct.